website/docs/d/private_link_service.html.markdown

```diff
@@ -0,0 +1,133 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_private_link_service"
+sidebar_current: "docs-azurerm-datasource-private-link-service"
+description: |-
+  Gets information about an existing Private Link Service
+---
+
+# Data Source: azurerm_private_link_service
+
+Use this data source to access information about an existing Private Link Service.
+
+
+## Private Link Service Usage
+
+```hcl
+data "azurerm_private_link_service" "example" {
+  resource_group_name = "acctestRG"
+  name                = "acctestpls"
```

Comment:
and here
```suggestion
  name                = "ThePrivateLink"
```

Reply:
fixed

Reply:
This is still using the acctest name

Reply:
My bad, this is for the data source I was looking at the resource documentation.